lib/ecto_crux.ex

```diff
@@ -150,489 +164,545 @@ defmodule EctoCrux do
       @doc "value of default select"
       def unquote(:select)(), do: @select
 
+      @doc "value of except"
+      def unquote(:except)(), do: @except
+
       @doc "create a new query using schema module"
       # eq: from e in @schema_module
       def init_query(), do: @init_query
 
       unless @read_only do
-        @doc "call schema_module changeset method"
-        def unquote(:change)(blob, attrs \\ %{}), do: @schema_module.changeset(blob, attrs)
+        unless excluded?(@except, :change, 2) do
```

Comment:
to avoid having a big block of unless @ready_only, we could simply do the following: if @read_only is true, we add all write functions in @except

Reply:
done